net-misc/sshpass/sshpass-1.09.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Tool for noninteractively performing password authentication with ssh"
+HOMEPAGE="https://sourceforge.net/projects/sshpass/"
+SRC_URI="mirror://sourceforge/${PN}/${PV}/${P}.tar.gz"
+LICENSE="GPL-2"
```

Comment:
Put LICENSE in the next block, so it's DESCRIPTION/HOMEPAGE/SRC_URI (new blank line) LICENSE/SLOT/KEYWORDS, and [drop blank IUSE](https://projects.gentoo.org/pms/8/pms.pdf#section.7.3).

RDEPEND is OK as-is.

Reply:
Done. Thanks for the link.